src/server/services/feature-flags.service.ts

```diff
@@ -66,8 +66,7 @@ const featureFlags = createFeatureFlags({
     description: `A helpful chat assistant that can answer questions about Stable Diffusion, Civitai, and more! We're still training it, so please report any issues you find!`,
     availability: ['mod', 'founder'],
   },
-  bounties: ['mod'],
-  modelMonetization: ['dev'],
+  bounties: ['dev'],
```

Comment:
I guess it can stay as mod for it to work on the dev site
```suggestion
  bounties: ['mod'],
```